.github/workflows/pr.yaml

```
@@ -32,11 +32,10 @@ jobs:
   test-conda-nightly-env:
     needs: checks
     secrets: inherit
-    # TODO: We want to use a build workflow so that we get CPU jobs and high
-    # matrix coverage without having to hardcode the CUDA/Python matrix here
-    uses: rapidsai/shared-workflows/.github/workflows/custom-job.yaml@branch-25.06
+    # We use a build workflow so that we get CPU jobs and high matrix coverage
+    uses: rapidsai/shared-workflows/.github/workflows/conda-python-build.yaml@optional-uploads
```

```suggestion
    uses: rapidsai/shared-workflows/.github/workflows/conda-python-build.yaml@branch-25.06
```

Plan:

1. see CI pass here
2. merge https://github.com/rapidsai/shared-workflows/pull/340
3. revert this back to `branch-25.06`
4. merge this PR